src/ms.tex

```diff
@@ -874,32 +874,32 @@ \subsection{Project funding} \label{sec:project-funding}
 % Summarize funding sources (Moore, NASA) and amounts and what this has been
 % used for.
 
-With the goal of establishing a sustainable financial model for the Astropy
-Project, the Coordination Committee, under the new governance charter (see
+With the goal of establishing a sustainable financial model for the \astropy
+Project, the CoCo, under the new governance charter (see
 Section~\ref{sec:project-governance}), established a standing Finance Committee.
 This committee oversees the planning and allocation of finances on behalf
-of the Project. The Astropy Project accepts funds from institutions as well as
+of the Project. The \astropy Project accepts funds from institutions as well as
 individuals through NumFOCUS.\footnote{NumFOCUS is a 501(c)(3) nonprofit that
 supports and promotes world-class, innovative, open-source scientific computing.}
-Previously, no direct financial support was available for the project development,
+Previously, no direct financial support was available for project development,
 and NumFOCUS covered most of the incurred operational costs. However, transitioning
 to long-term financial stability meant having an influx of funds in the form of
-institutional support, and hence the Astropy Project applied for and successfully
+institutional support, and hence the \astropy Project applied for and successfully
 acquired four grants over the last several years. These grants are administered by NumFOCUS on behalf of the
-Project, providing a ``neutral'' space not tied to any particular astronomy
+Project, providing a ``neutral'' space not tied to any particular astronomical
```

Comment:
I'm not sure about this word change. Instinctively, I want to read this as an adjective. I'm not sure we'd do the same if, for instance, it was "math" instead of "astronomy" (i.e. "any particular mathematical institution" vs. "any particular math institution").

Reply:
Agreed
```suggestion
Project, providing a ``neutral'' space not tied to any particular astronomy
```